app/database/models.py

```diff
@@ -297,3 +297,16 @@ def insert_data(target, session: Session, **kw):
 
 
 event.listen(Language.__table__, 'after_create', insert_data)
+
+
+class Note(Base):
+    __tablename__ = "notes"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String, nullable=False)
+    description = Column(String)
+    timestamp = Column(DateTime, index=True,
```

Comment:
Every note will propably have a differenat time, right?
Why you want to index by this field?

Reply:
Thanks, you are correct, I'll remove the index